src/transformers/modeling_utils.py

```diff
@@ -2431,6 +2431,11 @@ def from_pretrained(
                 commit_hash = getattr(config, "_commit_hash", None)
 
         if is_peft_available():
+            if adapter_kwargs is None:
+                adapter_kwargs = {}
```

Comment:
why not default adapter_kwargs to {}? Since you are not using something like `self.adapater_kwargs if adapter_kwargs is None else adapter_kwargs` doesn't make sense no?

Reply:
here yes it could be done